### PR TITLE
South America's Political Parties Overhaul

### PR DESCRIPTION
**Pull request by ColdSlav**
Added most political parties and political organizations to all the countries in South America
If I missed a party/org or don't agree with a party's policies/ideology make sure to tell me

### CODE REVIEW

**Comment by ghost** [comment]
How tf the Partido Radical is anarcho? Is stupid, the Partido Radical is a social democrats in fact hes official name is "Partido Radical Social Democrata" ( im talking about chile)

**Comment by ghost** [comment]
But i think is very good this commit

**Comment by ghost** [comment]
Just have some mistakes

**Comment by ColdSlav** [comment]
> How tf the Partido Radical is anarcho? Is stupid, the Partido Radical is a social democrats in fact hes official name is "Partido Radical Social Democrata" ( im talking about chile)

Anarcho-Liberals were changed and renamed to Radicals, as Anarcho-Liberals weren't really a thing back in the 19th Century, so, on the coming months we'll change Radicalism to fit more the 19th Century Radicals
According to what I researched and know, the "Partido Radical" (As the name says, Radical Party) was the result of the spread of European radical ideas to Latin America, starting with the Sociedad de la Igualdad in 1851.
So keep in mind they're not "Anarcho-Liberals" they're just named like that because that's how they were coded originally.
Also: Keep in mind that the term Social-Democrat has changed over time, so I don't think a late 20th Century SocDem is the same as an early 20th Century SocDem.

**Comment by ghost** [comment]
Dont put the partido radical so

**Comment by ColdSlav** [comment]
@RodriPosYo I added a socialist faction to the Partido Radical, we good now?

File: .gitignore [hunk 1]
```diff
@@ -53,4 +53,8 @@ AOE-exp/
 !/TGC-HFM-Style-Culture-Colors/
 !TGC-HFM-Style-Culture-Colors.mod
 !/TGC-Statehood-Mechanic/
-!TGC-Statehood-Mechanic.mod
\ No newline at end of file
+!TGC-Statehood-Mechanic.mod
+TGC/map/rivers.bmp
```
Comment (anzu-3278) [inline .gitignore]: why

**Comment by anzu-3278** [comment]
Apart from the .gitignore edit (which I assume is an accident, idk what river.bmp has with political parties), this looks okay - revert that file and feel free to merge.

File: .gitignore (hunk 1 above)
Comment (ColdSlav) [reply .gitignore]: > Apart from the .gitignore edit (which I assume is an accident, idk what river.bmp has with political parties), this looks okay - revert that file and feel free to merge.

Yeah that's an accident (Both gitignore and rivers lol) I was editing the rivers.bmp file to test how the river editing thing worked, but I didn't want the file to be included in the PR so I tried to make GitHub ignore the file (I thought it ignored the rivers.bmp so I didn't pay much attention to the PR when making it)

I guess that's how I messed up and ended up with both the Rivers.bmp and the gitignore files into the PR.
Give me a moment and I'll fix it

**Comment by ColdSlav** [comment]
> Apart from the .gitignore edit (which I assume is an accident, idk what river.bmp has with political parties), this looks okay - revert that file and feel free to merge.

I think I fixed it, should be good for merge now.

**Pull request merged**